### PR TITLE
Add documentation for update order endpoint

### PR DESCRIPTION
**Pull request by mvdpanne**
- Added a response example to the create order endpoint documentation.
- Minor fix to the update subscription endpoint documentation.

### CODE REVIEW

File: source/reference/v2/orders-api/create-order.rst [hunk 1]
```diff
@@ -588,3 +588,139 @@ Request (PHP)
          ]
        ]
    ]);
+
+Response
+^^^^^^^^
```
Comment (ibudasov) [inline source/reference/v2/orders-api/create-order.rst]: Don't you want say here something like: ....order object as described in `Get order`.

File: source/reference/v2/orders-api/update-order.rst [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+   $order->billingAddress->title = "Dhr";
+   $order->billingAddress->givenName = "Adriaan";
+   $order->billingAddress->familyName = "Mol";
+   $order->billingAddress->email = "adriaan@mollie.com";
+   $order->billingAddress->phone = "+31208202070";
+   $order->update();
+
+Response
+^^^^^^^^
+.. code-block:: http
```
Comment (ibudasov) [inline source/reference/v2/orders-api/update-order.rst]: same here: isn't just a link simpler?

Comment (mvdpanne) [reply source/reference/v2/orders-api/update-order.rst]: I was confused about this at first as well, but it seems that throughout the documentation, we have both a `Response` bit that mentions that links to another page, and a `Response` bit that shows an example response.

So to be clear: that link is actually in the file, but you'll have to fully open the file to be able to see it.

File: source/reference/v2/orders-api/update-order.rst [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+           "billingAddress": {
+               "streetAndNumber": "Keizersgracht 313",
+               "city": "Amsterdam",
+               "region": "Noord-Holland",
+               "postalCode": "1234AB",
+               "country": "NL",
+               "title": "Dhr",
+               "givenName": "Adriaan",
+               "familyName": "Mol",
+               "email": "adriaan@mollie.com",
```
Comment (willemstuursma) [inline source/reference/v2/orders-api/update-order.rst]: Don't do this!

Comment (mvdpanne) [reply source/reference/v2/orders-api/update-order.rst]: I more or less copy pasted this from the create order documentation, didn't know it was a problem.

I will fix it in both endpoints.

**Pull request merged**